run_tests.sh

```diff
@@ -3,10 +3,7 @@
 docker run --rm \
            -e USERNAME="Ipatios Asmanidis" \
            --net=host \
-           -v "$PWD/output":/output \
```

Comment:
This change breaks existing behaviour (existing output, scripts, reports directories are not longer used), you could just map them to /home/tester/output, /home/tester/scripts, /home/tester/reports.

For backwards compatiblity you may also want to add symlinks for the old directories:
/output -> /home/tester/output
/suites -> /home/tester/suites
/scripts -> /home/tester/scripts
/reports -> /home/tester/reports